app/services/food_service.rb

```diff
@@ -0,0 +1,14 @@
+class FoodService
+  class << self
+    def conn
+      Faraday.new(url: "https://api.nal.usda.gov") do |faraday|
+        faraday.params["api_key"] = ENV['food_api_key']
+      end
+    end
+
+    def search_for_food(query)
+      response = conn.get("/fdc/v1/foods/search?query=#{query}&dataType=&pageSize=10&pageNumber=1")
```

Comment:
It would be better to make the `pageSize` param dynamic:
```
def search_for_food(query, page_size = 10)
  response = conn.get("/fdc/v1/foods/search?query=#{query}&dataType=&pageSize=#{page_size}&pageNumber=1")
  ...
end
```

Personally, I would prefer to take the `pageSize` param out of this call altogether, in favor of having the facade handle it.